sdks/python/apache_beam/runners/worker/worker_id_interceptor.py

```diff
@@ -48,17 +49,23 @@ def __init__(self, worker_id=None):
     if worker_id:
       self._worker_id = worker_id
 
+  def intercept_unary_unary(self, continuation, client_call_details, request):
```

Comment:
You might as well and implement the UnaryStreamClientInterceptor and StreamUnaryClientInterceptor now.